README.md

```diff
@@ -90,14 +90,13 @@ This code of conduct and its related procedures also applies to unacceptable beh
 
 ## Contact info
 
-Feel free to contact any of the NewHaven.io board members at any time. You'll likely get a faster response on Slack than by email.
+Feel free to contact any of the NewHaven.io board members at any time. You'll likely get a faster response on Discord than by email.
 
 Henry Quinn | @henry | henry@newhaven.io
-Jenn Nolte | @Jenn Nolte she/her | jenn@newhaven.io
 Josh Levinson | @jlevnhv | josh@newhaven.io
-Steve Voils | @steve | steve@newhaven.io
 Tom Reznick | @tom | tom@newhaven.io
 Ben Midyette | @ben | ben@newhaven.io
+Grace de la Mora |
```

Comment:
I don't have contact info to put yet so TBD

Reply:
Updated these, I think we're all set with this PR. Thanks for putting it together, Grace!